site/en/docs/crux/api/index.md

```diff
@@ -0,0 +1,592 @@
+
+### URLs
+
+When the identifier is a url only data for that specific url will be returned. Looking again to the `http://www.example.com` origin sitemap:
+
+```text
+http://www.example.com
+http://www.example.com/foo.html
+http://www.example.com/bar.html
+```
+
+If the identifier is set to url with the value of `http://www.example.com/foo.html`, only data for that page will be returned.
+
+### Dimensions
+
+Dimensions identify a specific group of data that a record is being aggregated against, e.g. a form factor of Mobile indicates that the record contains information about loads that took place on a mobile device. Each dimension will have a certain number of values, and implicitly the lack of specifying that dimension will mean that the dimension is aggregated over all values. For example, specifying no form factor indicates that record contains information about loads that took place on any form factor.
+
+#### Form Factor
+
+The device class that the end-user used to navigate to the page. This is a general class of device split into `Phone`, `Tablet`, and `Desktop`. The `formFactor`
+
+### Metric
+
+See [metrics](../methodology/#metrics) for more info about the kinds of metrics included in the CrUX dataset.
+
+Metrics are expressed in a histogram, which represents the percent of users that experienced a metric with that value proportionally to all.
+
+A simple three bin histogram for metric "ExampleMetric" looks like this.
+
+```json
+{
+  "histogram": [
+    {
+      "start": 0,
+      "end": 1000,
+      "density": 0.38179
+    },
+    {
+      "start": 1000,
+      "end": 3000,
+      "density": 0.49905
+    },
+    {
+      "start": 3000,
+      "density": 0.11916
+    }
+  ],
+}
+```
+
+This data indicates that 38.2% of users experience an ExampleMetric value between 0ms and 1,000ms. The units of ExampleMetric are not contained in this histogram, in this case we will assume milliseconds.
+
+Additionally, 49.9% of users experience an ExampleMetric value between 1,000ms and 3,000ms, and 11.9% of users experience an ExampleMetric value greater than 3,000ms.
+
+Metrics will also contain percentiles that can be useful for additional analysis.
+
+```json
+{
+  "percentiles": {
+    "p75": 2063
+  },
+}
+```
+
+These percentiles can show specific metric values at the given percentile for that metric. They are based on the full set of available data and not the final binned data, so they do not necessarily match an interpolated percentile that is based on the final binned histogram.
+
+{% Aside %}
+Note: The values for each percentile are synthetically derived, it does not imply that any user actually experienced the value indicated, only that some percentage of users experienced a metric value that was less than the value given.
+{% endAside %}
+
+### Metric value types
+
+| CrUX API Metric Name                     | Data Type                | Metric Units | web.dev Docs                  |
```

Comment:
Please use the table markup for all tables https://developer.chrome.com/docs/handbook/components/#tables